lib/modules/redis/redis.go

```diff
@@ -0,0 +1,33 @@
+package redis
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/go-redis/redis"
+)
+
+type Redis struct {
+	Client *redis.Client
+}
+
+var sharedInstance = New()
+
+func New() *Redis {
+	host := os.Getenv("REDIS_HOST")
+	port := os.Getenv("REDIS_PORT")
+	number := os.Getenv("REDIS_DB_NUMBER")
+	i, _ := strconv.Atoi(number)
+	client := redis.NewClient(&redis.Options{
+		Addr:     host + ":" + port,
+		Password: "", // no password set
+		DB:       i,
+	})
+	return &Redis{
+		client,
+	}
+}
+
+func SharedInstance() *Redis {
```

Comment:
__[Golint]__ exported function SharedInstance should have comment or be unexported ([link](https://sider.review/gh/repos/50192141/pulls/456#issue-3827453))

<sub>posted by Sider</sub>